### PR TITLE
Refactored code a little and centered the card grid

### LINKED ISSUE #9: Size for display
Activity does not fit to display size.  For example, on an Ubuntu 18.04 KVM with 1024x768 4:3 display;

screenshot_ubb_2018-05-08_10 23 38

For the activity to fit to display size;
* at minimum, the orange section should be centred in the window,
* at best, the orange section should be enlarged to cover more pixels within the window.

The orange section includes the contents of the section.

### PR DESCRIPTION
**Pull request by marsian83**
There were a lot of functions for the same task btu for different grid sizes I've added a few properties to reduce these.
this has been mainly done to easily implement the centered card grid, The cards were not centered properly to the screen, this fixed it. 

Also, even though I did not initially intend for this to be, but now it is kind of a solution to #9 

Fixes #9.

### CODE REVIEW

File: grid.py [hunk 1]
```diff
@@ -214,6 +201,20 @@ def print_orientations(self):
 
     # Test all relevant borders, ignoring edges
     # Highlight matches.
+    def test(self):
```
Comment (chimosky) [inline grid.py]: Could you rename this to `test_borders` as the comment right above it says it tests borders.

**Comment by chimosky** [comment]
Tested d2af304, works as expected.

I'll wait for you to make the change above before I merge.

**Comment by marsian83** [comment]
Hi @chimosky , Are we waiting for any further changes with this one?

**Comment by chimosky** [comment]
Thanks for bringing this to my attention.

**Pull request merged**